ETC.tsv

```diff
@@ -7062,10 +7062,10 @@ ETC_20150317_007450	Suspicious spell device
 ETC_20150317_007451	Activate spell device
 ETC_20150317_007453	Hire Mercenary!
 ETC_20150317_007454	Protect big merchant Gilliam
-ETC_20150317_007455	There is a news that a big merchant Gilliam who did not sell items to any person has been placed at a crossroad of new city.
+ETC_20150317_007455	There are news that Gilliam, a big merchant who did not sell items to anyone, has been placed at a crossroad of a new city.
 ETC_20150317_007456	Basic Session
 ETC_20150317_007457	Quest Hitme Reward
-ETC_20150317_007458	Run away from monster safe area
+ETC_20150317_007458	Escape from the monster refuge!
```

Comment:
The monster isn't a refuge. This line seems to be indicating to get to a safe area away from the monster. So "Run away from the monster to a safe area" seems more appropriate